deployment/examples/kustomization_storage_account.yaml

```diff
@@ -17,10 +22,12 @@ patches:
     version: v1
   patch: |-
     - op: add
-      path: '/spec/template/spec/containers/0/env/-'
+      path: '/spec/template/spec/containers/0/env'
       value:
-        name: AZURE_BLOB_ACCOUNT_NAME
-        value: <name>
+        - name: AZURE_BLOB_ACCOUNT_NAME
+          value: <name>
+        - name: AZURE_BLOB_CONTAINER_NAME
```

Comment:
Currently our CLI code calls the aks-periscope.yaml file https://github.com/alphaWizard/azure-cli-extensions/blob/5dd22cbf7cf03e26f374a62504c90a5403bdd1c7/src/connectedk8s/azext_connectedk8s/_utils.py#L757. To avoid a large refactoring of the CLI code, is it possible add the AZURE_BLOB_CONTAINER_NAME field to the single yaml file as well?